app/models/profile.rb

```diff
@@ -0,0 +1,9 @@
+class Profile < ApplicationRecord
+
+  belongs_to :user
+
+  def name
+    first_name + " " + last_name
+  end
+
```

Comment:
You should add some validations to define what is permitted in someone's profile. For example, I was able to make my gender Lion!